packages/components/src/components/Radio/RadioGroup.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import {
+  Root as RadioGroupRoot,
+  RadioGroupProps as RadioGroupPrimitiveProps,
+} from "@radix-ui/react-radio-group";
+import { Box } from "../../primitives/Box";
+
+export interface RadioGroupProps
+  extends Omit<RadioGroupPrimitiveProps, "nonce"> {
```

Comment:
I don't think we need to omit `nonce` here. Thats just coming from Ariakit primitives.